### PR TITLE
blog: December 2024 newsletter

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

December 2024 newsletter, rendered at https://20241203-december-newsletter.ddev-com-front-end.pages.dev/blog/ddev-dec-2024-newsletter/

Minor update to release blog

### CODE REVIEW

**Review by rpkoller — APPROVED** [review]
looks good to me. the sole super tiny nitpick would be the term `release blog`. for me, my first association is, there is an entire blog (with several posts) for that release. ^^ personally i would rather choose `blog post` instead. but as i said super nitpicky and based on my own perception as a none native speaker.

File: src/content/blog/ddev-dec-2024-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+featureImage:
+  src: /img/blog/2024/12/nancy_lewis_colorado_river.jpg
+  alt: Nancy Lewis painting of Colorado River near Palisade, Colorado
+categories:
+  - Community
+---
+
+**Hello DDEV Community!** It’s always a pleasure to reach out to you at the beginning of the month. Here are some updates for you.
+
+**DDEV v1.24.0** was released with lots of goodies, including the full PHP 8.4.1 with all extensions, including Xdebug. This is a major release with some deprecations and removals, please read the [release blog](release-v1.24.0-php8.4.md) and [release notes](https://github.com/ddev/ddev/releases/tag/v1.24.0).
```
Comment (tyler36) [inline src/content/blog/ddev-dec-2024-newsletter.md:16]:
```suggestion
**DDEV v1.24.0** was released with lots of goodies, including PHP 8.4.1 with all extensions, as well as Xdebug. This is a major release with some deprecations and removals, please read the [release blog](release-v1.24.0-php8.4.md) and [release notes](https://github.com/ddev/ddev/releases/tag/v1.24.0).
```

File: src/content/blog/ddev-dec-2024-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+**TYPO3 Community Budget Ideas:** DDEV has posted a community budget idea for TYPO3's [community budget process](https://typo3.org/article/budget-2025-ideas-for-quarter-1-2025-published-vote-now). We'd like to improve DDEV's xhprof and xhgui even beyond where [@tyler36](https://github.com/tyler36) has already taken the [DDEV xhgui add-on](https://github.com/ddev/ddev-xhgui) and incorporate it into core. If you're a member of the TYPO3 association you've gotten an email about voting on this, please vote for the DDEV proposal!
+
+**DDEV Blogs and News**
+
+* [DDEV Usage Statistics](stats-on-ddev-usage-nov-2024.md)
+* [Setting up a Windows Machine for DDEV](windows-ddev-setup.md)
+* [Contributor Training: DDEV Analytics with Amplitude](https://ddev.com/blog/amplitude-ddev-analytics-contributor-training/)
+* [DDEV Advisory Group Meeting](https://github.com/orgs/ddev/discussions/6682). These meetings happen bimonthly and you're always invited. The next one is Wednesday, January 8, 2025, and is the most important meeting of the year. We will review 2024 and plan for 2025. We'd love to have you [join us](https://www.meetup.com/ddev-events/events/303197425).
```
Comment (tyler36) [inline src/content/blog/ddev-dec-2024-newsletter.md:25]:
```suggestion
* [DDEV Advisory Group Meeting](https://github.com/orgs/ddev/discussions/6682). These meetings happen every two months and you're always invited. The next one is Wednesday, January 8, 2025, and is the most important meeting of the year. We will review 2024 and plan for 2025. We'd love to have you [join us](https://www.meetup.com/ddev-events/events/303197425).
```

"bimonthly" means "twice a month or every two months" so we should be more specific.

File: src/content/blog/ddev-dec-2024-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+**THANKS to all of you who are supporting DDEV’s path to sustainability** and who have gotten your organizations to do so.
+
+Want to keep up as the month goes along? Follow our [blog](https://ddev.com/blog/), [LinkedIn](https://www.linkedin.com/company/ddev-foundation), [Mastodon](https://fosstodon.org/@ddev), and join us on [Discord](https://discord.gg/5wjP76mBJD).
```
Comment (tyler36) [inline src/content/blog/ddev-dec-2024-newsletter.md:40]:
```suggestion
Want to keep up as the month goes along? Follow our
* [blog](https://ddev.com/blog/), 
* [LinkedIn](https://www.linkedin.com/company/ddev-foundation), 
* [Mastodon](https://fosstodon.org/@ddev), 
* [BlueSky](https://bsky.app/profile/ddev.bsky.social) 
* and join our community on [Discord](https://discord.gg/5wjP76mBJD).
```

Added BlueSky
Slight formatting might improve brand recognition for visual scanner?
Highlight the community hangs out at Discord

**Review by tyler36 — CHANGES_REQUESTED** [review]
Some minor suggestions and a recommendation against "bimonthly"

File: src/content/blog/ddev-dec-2024-newsletter.md (hunk 3 above)
Comment (rfay) [reply src/content/blog/ddev-dec-2024-newsletter.md:40]: Thanks!

File: src/content/blog/ddev-dec-2024-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+**THANKS to all of you who are supporting DDEV’s path to sustainability** and who have gotten your organizations to do so.
+
+Want to keep up as the month goes along? Follow our
+* [blog](https://ddev.com/blog/), 
+* [LinkedIn](https://www.linkedin.com/company/ddev-foundation), 
+* [Mastodon](https://fosstodon.org/@ddev), 
+* [BlueSky](https://bsky.app/profile/ddev.bsky.social)
```
Comment (stasadev) [inline src/content/blog/ddev-dec-2024-newsletter.md:44]:
```suggestion
* [BlueSky](https://bsky.app/profile/ddev.bsky.social),
```

**Review by stasadev — APPROVED** [review]

**Pull request merged**